test/gateway/eip20_gateway/revert_stake.js

```diff
@@ -52,9 +66,289 @@ contract('EIP20Gateway.revertStake()', function (accounts) {
       burner,
     );
 
+    let hashLockObj = Utils.generateHashLock();
+
+    stakeMessage.hashLock = hashLockObj.l;
+    stakeMessage.unlockSecret = hashLockObj.s;
```

Comment:
`l` and `s` are horrible field names 😱

Reply:
Yes, agree!!! This is used across files, my IDE is not smart enough to refactor this easily 😞  

Can we please handle this in a separate ticket?

Reply:
Of course. Just wanted to mention it.

Reply:
Create ticket 
https://github.com/OpenSTFoundation/mosaic-contracts/issues/597